context.go

```diff
@@ -274,13 +275,22 @@ func (c *Context) MustGet(key string) any {
 }
 
 // GetString returns the value associated with the key as a string.
-func (c *Context) GetString(key string) (s string) {
+func (c *Context) GetString(key string) (s string, err error) {
```

Comment:
This is an adjustment that is not backward compatible, and it is not recommended to make such an adjustment.